dojo/pipeline.py

```diff
@@ -86,6 +86,16 @@ def update_product_access(backend, uid, user=None, social=None, *args, **kwargs)
                 product, created = Product.objects.get_or_create(name=project_name, prod_type=product_type)
                 if not settings.FEATURE_AUTHORIZATION_V2:
                     product.authorized_users.add(user)
+                    # Add tags if needed
+                    if settings.GITLAB_PROJECT_IMPORT_TAGS:
+                        tags = [",".join(project.tag_list) for project in projects if project.path_with_namespace == project_name]
```

Comment:
Does a project always have something in the tag_list field? I'm wondering if the default is at least `[]` so that the join function does not break.

Reply:
Not sure that there is always a tag_list field, so I will check. And `tag_list` seems to be deprecated, replaced by `topics`, so I will first look for topics in priority.